From 5717b3a12ed7df158abf89fc79d030c415c1a113 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 16 Jul 2025 12:31:33 +0100
Subject: [PATCH] ublk: Remove unused EXPECTED_VERSION

Probably we should test nbdublk --version.  As we do not, this
variable was not used.

(cherry picked from commit 01f5d93d43f7eab0444c87d9d99e2ecea9bf9d44)
---
 ublk/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ublk/Makefile.am b/ublk/Makefile.am
index 667d7d0c..e06e4396 100644
--- a/ublk/Makefile.am
+++ b/ublk/Makefile.am
@@ -24,7 +24,6 @@ EXTRA_DIST = \
 TESTS_ENVIRONMENT = \
 	LIBNBD_DEBUG=1 \
 	$(MALLOC_CHECKS) \
-	EXPECTED_VERSION=$(VERSION) \
 	$(NULL)
 LOG_COMPILER = $(top_builddir)/run
 TESTS =
-- 
2.47.3

